webpack/sass/main.scss

```diff
@@ -1,3 +1,5 @@
+@import "../node_modules/@creativecommons/vocabulary/css/vocabulary.css";
```

Comment:
You're including vocabulary twice? I mean... it's already on the header right?

Reply:
OOPSIE

Reply:
I updated it!